### ISSUE TITLE
BMC and hybrid journals until 2015

### ISSUE THREAD

**Issue by adrsik** [OpenAPC/openapc-de#71]
Since 2015 BioMed Central no longer requires a subscription to access additional content (reviews and other permium contenct (commentaries, correspondence, meeting reports, ...)  of the following journals:

* Alzheimer's Research & Therapy
* Arthritis Research & Therapy
* Breast Cancer Research
* Critical Care
* Genome Medicine
* Genome Biology
* Stem Cell Research & Therapy 

So all articles published bevore 2015 of these journals should be hybrid?
I noticed several examples in the data set which is not marked as a hybrid article. 
Is there a special reason?

**Comment by tullney** [comment]
@adrsik , I tend to agree. The journals in questions required subscriptions to access particular sections of the journal - so they weren't hybrid in the usual way of pay-per-article hybrid APC, but they weren't full open access journals as in "every article in this journal is open access".

We could compile a list of these articles and change their status to hybrid, but maybe also talk to the institution that provided that data.

**Comment by cbroschinski** [comment]
I have just compiled an overview of the journals in question (table is already filtered to those entries where is_hybrid is still FALSE):

journal_full_title | occurence
------------------------ | -----------------
Alzheimer's Research & Therapy|0
Arthritis Research & Therapy|5
Breast Cancer Research|1
Critical Care|17
Genome Biology|4
Genome Medicine|1
Stem Cell Research & Therapy|3

So we are only talking about 31 entries in the whole dataset we would have to modify. Does not seem like a big deal to me.

**Comment by tullney** [comment]
If you added institution to your query, we'd also have a list of people to notify about changes in the data they submitted.

I am still in favour of doing this, but we should probably add a note to the wiki explaining that `is_hybrid` refers to the journal - not the individual article. Still it might be controversial. People investigating APCs in full open access journals and in 'hybrid' journals might disagree with framing these journals as hybrid.

Maybe leave this issue open for discussion for a few more days?
